pkg/apis/core/validation/cloudprofile.go

```diff
@@ -239,6 +241,12 @@ func validateMachineTypes(machineTypes []core.MachineType, fldPath *field.Path)
 		}
 		names[machineType.Name] = struct{}{}
 
+		if len(machineType.Architecture) == 0 {
+			allErrs = append(allErrs, field.Required(archPath, "must provide CPU architecture type of the machine"))
```

Comment:
Do we HAVE to enforce this? Can't we, to reduce maintenance effort, just assume that an empty list of `architectures` means that all architectures are supported? Otherwise, we have to add this for each and every single machine type...